README.md

```diff
@@ -0,0 +1,4 @@
+# sport-max
+
+
+The maximum of sport
```

Comment:
Where the heck is the final point ? Please, add it before merging.

Reply:
Good catch! fixed. Thank you.